lib/matplotlib/offsetbox.py

```diff
@@ -1053,13 +1055,22 @@ def set_bbox_to_anchor(self, bbox, transform=None):
 
     @_compat_get_offset
     def get_offset(self, bbox, renderer):
-        # docstring inherited
-        pad = (self.borderpad
-               * renderer.points_to_pixels(self.prop.get_size_in_points()))
+        fontsize_in_pixels = renderer.points_to_pixels(self.prop.get_size_in_points())
+        try:
+            borderpad_x, borderpad_y = self.borderpad
+        except (TypeError):
```

Comment:
```suggestion
        except TypeError:
```
Brackets are not needed if we only catch one exception.